js/app.js

```diff
@@ -0,0 +1,89 @@
+document.addEventListener(`DOMContentLoaded`, () =>{
+
+	const form = document.querySelector(`#registrar`);
+	const formInput = form.querySelector(`input`);
+	const invitedUl = document.querySelector(`#invitedList`);
+	const hideNotResponded = document.querySelector(`.main input`);
+
+	function newElement(elementParent, element) {
+		const name = document.createElement(element);
+		elementParent.appendChild(name)
+		return name
+	}
+
+	function text(elementParent, element, text) {
+		const name = newElement(elementParent, element);
+		name.textContent = text;
+		return name;
+	}
+
+	function type(elementParent, element, type) {
+		const name = newElement(elementParent, element);
+		name.type = type;
+		return name;
+	}
+
+	function submitForm() {
+		const newLi = newElement(invitedUl, 'li');
+		const newSpan = text(newLi, `span`, formInput.value);
+		const newInput = type(newLi, `input`, 'text');
```

Comment:
Podsumowanie tego o czym pisałem wyżej, masz 3 różne metody, które częściowo robią to samo. To wbrew DRY i taki kod będzie trudny do otestowania